### ISSUE TITLE
[Developers] [Module 6 Custom Widgets]

### ISSUE THREAD

**Issue by bakkenb** [w3c/wai-curricula#280]
[strong] (typo) Under topic Semantics and Widget Roles / Ideas to Assess... The following sentence has typo - "Practical — Students are presented with different custom widgets that do not contain accessibility properties and are asked to code them using standard HTML elements to the extend possible." Final words should be "... extent possible."

**Comment by daniel-montalvo** [comment]
Fixed at
http://deploy-preview-273--wai-curricula.netlify.app/curricula/developing-accessible-content/custom-widgets/#topic-semantics-and-widget-roles
